opentech/apply/review/tests/factories/blocks.py

```diff
@@ -0,0 +1,82 @@
+import json
+import random
+import uuid
+
+import factory
+from wagtail.core.blocks import CharBlock
+import wagtail_factories
+
+from opentech.apply.stream_forms import blocks as stream_blocks
+from opentech.apply.review import blocks
+from opentech.apply.utils import blocks as utils_blocks
```

Comment:
I would prefer to include these in the stream form factories and import rather than copy them across from the funds factories. (which i think has happened here).